security/rbac/docs/index.rst

```diff
@@ -447,6 +447,7 @@ User ``bender:bender`` doesn’t have any role bindings set up and can be used a
 -  You can use ``./client-configs/bender.properties`` file to authenticate as ``bender`` from kafka console commands (like ``kafka-console-producer``, ``kafka-console-consumer``, ``kafka-topics`` and the like).
 -  This file is also mounted into the broker docker container, so you can ``docker-compose -p [project-name] exec broker /bin/bash`` to open bash on broker and then use console commands with ``/etc/client-configs/bender.properties``.
 -  When running console commands from inside the broker container, use ``localhost:9092``.
+-  In this example, the ``bender`` client uses the token services for simplicity in this dev environment (e.g. bender.properties uses ``sasl.mechanism=OAUTHBEARER``). But in production, the client should use Kerberos, mTLS, or SASL/PLAIN for authentication
```

Comment:
You wouldn't want to use SASL/PLAIN in production so I'd take that out, Otherwise looks good to me